src/utils/poll.rs

```diff
@@ -22,9 +23,12 @@ pub async fn handle_poll(
 
     let eligible = member.joined_at.unwrap() <= min_join_date;
     let auth_token = if eligible {
+        let mut hasher = Sha3_256::new();
+        hasher.update(&member.user.id.to_string());
         let result = data
             .api
-            .authenticate(&member.user.id.to_string())
+            // Only use part of the hash so the final URL doesn't get too big.
+            .authenticate(&hex::encode(hasher.finalize())[..40])
```

Comment:
This is an arbitrary limit for no particular reason, if this is really required, it can be limited to 2^5 = 2 << 6 = 32 chars.

Reply:
The reason why we cannot use the entire hash is because the URL, which contains the PASETO token, would be above the 512 character limit imposed by the Discord components API.

32 makes more sense I guess, so I will use that instead.